app/src/main/res/layout/intro2.xml

```diff
@@ -2,36 +2,36 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:orientation="vertical" android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="#018e9f"
-    android:layout_weight="10"
+    android:background="@color/intro2_background"
+    android:layout_weight="@integer/intro4_layout_weight"
     android:id="@+id/main">
 
     <TextView
         android:layout_width="wrap_content"
-        android:layout_height="0dp"
+        android:layout_height="@dimen/zero"
         android:layout_gravity="center"
         android:gravity="center"
-        android:paddingLeft="32dp"
-        android:layout_weight="3"
+        android:paddingLeft="@dimen/intro_padding"
+        android:layout_weight="@integer/intro4_textView_weight"
         android:fontFamily="sans-serif-thin"
-        android:textColor="#ffffff"
-        android:paddingRight="32dp"
-        android:textSize="@dimen/text_large"
-        android:text="@string/circle_of_trust_slide_two"/>
+        android:textColor="@color/intro_text_color"
+        android:paddingRight="@dimen/intro_padding"
+        android:textSize="@dimen/text_normal"
+        android:text="@string/intro2_top_text"/>
 
```

Comment:
Basically what you have to do is add the values which are not already there in strings, integers , etc. Do not remove already added things. It may remove other people's changes